client/src/component/EventDialog/EventType.js

```diff
@@ -0,0 +1,164 @@
+import React from "react";
+import Grid from "@material-ui/core/Grid";
+import Button from "@material-ui/core/Button";
+import Box from "@material-ui/core/Box";
+import {
+  createMuiTheme,
+  withStyles,
+  makeStyles,
+  ThemeProvider,
+} from "@material-ui/core/styles";
+import Icon from "@material-ui/core/Icon";
+import { grey, orange } from "@material-ui/core/colors";
+import { Typography, Container } from "@material-ui/core";
+import CreateEventDialog from "./CreateEventDialog";
+
+const useStyles = makeStyles((theme) => ({
+  margin: {
+    margin: theme.spacing(1),
+  },
+  title: {
+    marginTop: 30,
+    marginBottom: 40,
+  },
+
+  icon_grid: {
+    height: "5rem",
+    width: "5rem",
+  },
+  options: {
+    marginTop: "4rem",
+  },
+}));
+
+const defaultProps = {
+  style: { width: "5rem", height: "5rem" },
+  border: 1,
+};
+
+const theme = createMuiTheme({
+  palette: {
+    primary: {
+      main: orange[500],
+    },
+    secondary: {
+      main: grey[700],
+    },
+  },
+});
+
+const setTypeOne = () => {
+  localStorage.removeItem("type");
+  localStorage.setItem("type", "One-on-one");
+};
+
+const setTypeGroup = () => {
+  localStorage.removeItem("type");
+  localStorage.setItem("type", "Group");
+};
```

Comment:
These should either be in state or if they're from a parent component they should be passed down as a prop